src/main/java/duke/application/Application.java

```diff
@@ -0,0 +1,149 @@
+package duke.application;
+
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.message.Message;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import javafx.application.Platform;
+import javafx.stage.Stage;
```

Comment:
Perhaps there should be a new line in between for clarity in importing?

```suggestion
import duke.ui.Ui;

import javafx.application.Platform;
import javafx.stage.Stage;
```